package.json

```diff
@@ -25,6 +25,7 @@
   "devDependencies": {
     "flush-write-stream": "^2.0.0",
     "from2": "^2.1.1",
+    "null-writable": "^1.0.5",
```

Comment:
@mcollina please don't pull in this dependency. It has ~217 weekly downloads and I've never heard of the author. I'm sure from2 or core streams can be used instead.

Reply:
Removed the dependency and internalized its code. It's MIT-licensed.